config/protocol_config.go

```diff
@@ -38,11 +38,11 @@ func (c *ProtocolConfig) Prefix() string {
 }
 
 func loadProtocol(protocolsIds string, protocols map[string]*ProtocolConfig) []*ProtocolConfig {
-	returnProtocols := []*ProtocolConfig{}
+	var returnProtocols []*ProtocolConfig
```

Comment:
```suggestion
	returnProtocols :=make([]*ProtocolConfig,0,len(protocols))
```
Specify the capacity and length is better

Reply:
Nice advice. I specify the capacity when initializing slice.